frontend/test/metabase/scenarios/models/reproductions/20624-model-metadata-should-override-column-settings.cy.spec.js

```diff
@@ -27,9 +30,9 @@ describe.skip("issue 20624", () => {
     cy.findByText("Customize metadata").click();
 
     // Open settings for this column
-    cy.findByText(renamedColumn).click();
+    cy.findByText(originalColumn).click();
```

Comment:
I need to change the repro a bit. See my comment here https://github.com/metabase/metabase/issues/20624#issuecomment-1070849035

Let me know if you think it should behave otherwise.